jest/jest-base.config.js

```diff
@@ -29,9 +29,10 @@ module.exports = {
     '^@asap-hub/([^/]+)$': '@asap-hub/$1/src',
   },
   modulePathIgnorePatterns: [
-    regexpEscape(root) + '(/(apps|packages)/[^/]+)?/build(-cjs)?/',
-    regexpEscape(root) + '/coverage/',
-    regexpEscape(root) + '/packages/services-common/src/cms/',
+    'build(-cjs)?',
```

Comment:
this should be sufficient - the problem on windows is using backslash vs slash
I don't think we will ever have tests in subdirectories of build* or coverage, right?